iron-overlay-behavior.html

```diff
@@ -11,8 +11,8 @@
 <link rel="import" href="../polymer/polymer.html">
 <link rel="import" href="../iron-fit-behavior/iron-fit-behavior.html">
 <link rel="import" href="../iron-resizable-behavior/iron-resizable-behavior.html">
-<link rel="import" href="iron-overlay-backdrop.html">
 <link rel="import" href="iron-overlay-manager.html">
+<link rel="import" href="iron-overlay-backdrop.html">
```

Comment:
would you mind reverting this as before? :angel: 
(since there is no particular reason to be moved as last import)